src/library/scala/collection/IterableOnce.scala

```diff
@@ -43,37 +43,37 @@ final class IterableOnceExtensionMethods[A](private val it: IterableOnce[A]) ext
     case _ => it.iterator.foreach(f)
   }
 
-  @deprecated("Use factory.from(it) instead of it.to(factory) for IterableOnce", "2.13.0")
+  @deprecated("Use it.to(factory) instead of factory.from(it) for IterableOnce", "2.13.0")
```

Comment:
Have you tried that syntax? (`it.to(factory)`)

I suspect that you will get the deprecation warning. I think the syntax should be `it.iterator.to(factory)`.

Reply:
Ah alright, will change this